artifacts/live-airflow-provider-execution-20260322/BLOCKER.md

```
@@ -0,0 +1,42 @@
+# Blocker — Full Airflow Scheduler Execution (Issue #36)
+
+## Status
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)